--- "old/.pio/libdeps/esp32/ESP Async WebServer/src/WebAuthentication.cpp"
+++ "new/.pio/libdeps/esp32/ESP Async WebServer/src/WebAuthentication.cpp"
@@ -22,6 +22,8 @@
 #include <libb64/cencode.h>
 #ifdef ESP32
 #include "mbedtls/md5.h"
+
+
 #else
 #include "md5.h"
 #endif
@@ -71,9 +73,9 @@
   memset(_buf, 0x00, 16);
 #ifdef ESP32
   mbedtls_md5_init(&_ctx);
-  mbedtls_md5_starts(&_ctx);
-  mbedtls_md5_update(&_ctx, data, len);
-  mbedtls_md5_finish(&_ctx, _buf);
+  mbedtls_md5_starts_ret(&_ctx);
+  mbedtls_md5_update_ret(&_ctx, data, len);
+  mbedtls_md5_finish_ret(&_ctx, _buf);
 #else
   MD5Init(&_ctx);
   MD5Update(&_ctx, data, len);

